.github/workflows/freshen-images/build.sh

```diff
@@ -83,15 +83,23 @@ ansible-operator)
   # ansible-operator has a base image that must be rebuilt in advance if necessary.
   # This script will detect that the base is fresh when inspecting ansible-operator's
   # Dockerfile and build it.
-  for tag in $TAGS; do
-    build_ansible_base $tag "$PLATFORMS"
+  for i in ${!TAGS[*]}; do
+    if (($i=0)); then
```

Comment:
I haven't looked at the code for getting the tags, but is it the case that the latest tag will always be the first tag in the list?

Reply:
yes, the biggest number should always be first in the list